### PR TITLE
Add test for ListCommand with empty AddressBook

### LINKED ISSUE #112: Update ListCommandTest to ensure functionality when AddressBook is empty
Currently, the ListCommand is relatively simple and works fine, but there is no test to ensure that it still works with an empty AddressBook. We should add a test to ensure this command passes this check.

### PR DESCRIPTION
**Pull request by bipplane**
This PR will add a test to ensure ListCommand still works even with an empty AddressBook, for future code validation.

Fixes #112

### CODE REVIEW

File: src/test/java/seedu/address/logic/commands/ListCommandTest.java [hunk 1]
```diff
@@ -36,4 +37,13 @@ public void execute_listIsFiltered_showsEverything() {
         showPersonAtIndex(model, INDEX_FIRST_PERSON);
         assertCommandSuccess(new ListCommand(), model, ListCommand.MESSAGE_SUCCESS, expectedModel);
     }
+
+    @Test
+    public void execute_emptyAddressBook_showsEmptyList() {
+        // Create empty models
```
Comment (izniy) [inline src/test/java/seedu/address/logic/commands/ListCommandTest.java:43]: Good addition of comment to clearly indicate what the test case is doing

**Review by izniy — APPROVED** [review]
LGTM! The test helps to check for the case of an empty AddressBook

**Review by flljy940 — APPROVED** [review]
